GovUkHtmlHelperExtensions.cs

```diff
@@ -295,5 +296,23 @@ public static class GovUkHtmlHelperExtensions
                 classes,
                 textInputAppendix);
         }
+
+        public static IHtmlContent GovUkFileUploadFor<TModel>(
+            this IHtmlHelper<TModel> htmlHelper,
+            Expression<Func<TModel, IFormFile>> propertyExpression,
+            LabelViewModel labelOptions = null,
+            HintViewModel hintOptions = null,
+            FormGroupViewModel formGroupOptions = null,
+            string classes = null)
+            where TModel : class
+        {
+            return FileUploadHtmlGenerator.GenerateHtml(
+                htmlHelper,
+                propertyExpression,
+                labelOptions,
+                hintOptions,
+                formGroupOptions,
+                classes);
+        }

```

Comment:
Could you please also add a `GovUkFileUpload()` method? See `GovUkTextArea` and `GovUkTextAreaFor` for an example

Reply:
Done